pkg/cli/admin/upgrade/status/health.go

```diff
@@ -170,6 +170,23 @@ func assessUpdateInsights(insights []updateInsight, upgradingFor time.Duration,
 }
 
 func shortDuration(d time.Duration) string {
+	if d > time.Hour*24*365*2 {
+		y := d / (time.Hour * 24 * 365)
+		day := (d % (time.Hour * 24 * 365)) / (time.Hour * 24)
+		if day == 0 {
+			return fmt.Sprintf("%dy", y)
+		}
+		return fmt.Sprintf("%dy%dd", y, day)
+	} else if d > time.Hour*24*7 && d <= time.Hour*24*365*2 {
+		return fmt.Sprintf("%dd", d/(time.Hour*24))
+	} else if d > time.Hour*24*2 && d <= time.Hour*24*7 {
+		day := d / (time.Hour * 24)
+		hour := (d % (time.Hour * 24)) / time.Hour
+		if hour == 0 {
+			return fmt.Sprintf("%dd", d/(time.Hour*24))
+		}
+		return fmt.Sprintf("%dd%dh", day, hour)
+	}
```

Comment:
Like stated above, I do not think we need to handle days and years. We are a UI focused on the process that should take hours, or several days at max. I think it's totally fine to say `48h` instead of `2d` when such (pathological) condition occurs. It better communicates that the _intended_ duration unit is hours, and a high number communicates that a long duration is somehow not expected.

Reply:
Removed.